### PR TITLE
Update Pixi.js auto-update config w/ v4.2.1/4.2.2

### PR DESCRIPTION
**Pull request by bigtimebuddy**
The auto-updating stopped working because **pixi.js** changed the release structure. This fixes the **package.json** and adds missing versions 4.2.1 and 4.2.2.

### CODE REVIEW

File: ajax/libs/pixi.js/package.json [hunk 1]
```diff
@@ -15,10 +15,12 @@
...
   "npmFileMap": [
     {
-      "basePath": "bin",
+      "basePath": "dist",
       "files": [
         "pixi.js",
-        "pixi.dev.js"
+        "pixi.js.map",
+        "pixi.min.js",
+        "pixi.min.js.map"
```
Comment (PeterDaveHello) [inline ajax/libs/pixi.js/package.json]: We could just use single `"**/*"` here to grab all the files, would you please use this rule to replace all the others? Thanks.

**Comment by bigtimebuddy** [comment]
@PeterDaveHello done!

**Comment by PeterDaveHello** [comment]
@bigtimebuddy please still keep the indent as:
```json
      "files": [
         "**/*"
       ]
```

or you will not pass the test, thanks.

**Comment by bigtimebuddy** [comment]
@PeterDaveHello Fixed.

**Comment by PeterDaveHello** [comment]
@bigtimebuddy Thanks

File: ajax/libs/pixi.js/package.json [hunk 2]
```diff
@@ -16,7 +16,9 @@
   "npmFileMap": [
     {
       "basePath": "dist",
-      "files": ["**/*"]
+      "files": [
+         "**/*"
```
Comment (PeterDaveHello) [inline ajax/libs/pixi.js/package.json]: need to remove one more space here.

**Comment by bigtimebuddy** [comment]
@PeterDaveHello I think we're good now.

**Comment by PeterDaveHello** [comment]
@bigtimebuddy Yes we are! Thanks for your contribution!

**Pull request merged**